PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,8 +21,4 @@
 - [ ] `.changeset`
 - [ ] unit tests
 - [ ] integration tests
-- [ ] docs
```

Comment:
I thought `docs` means we should add or update docs in `https://docs.logto.io`

Reply:
yes, but the docs update has a really low frequency. would prefer to mention the TSDoc comments instead

Reply:
synced offline